### PR TITLE
Rewrite the install page for steam deck

### LINKED ISSUE #35: Steam screenshots still PolyMC
Also we might add a reference to `https://www.steamgriddb.com/boop` which makes installing artwork a lot less cumbersome when using steamgriddb

### PR DESCRIPTION
**Pull request by Lutitious**
This will encourage the user to use [sdgboop](https://www.steamgriddb.com/boop) which is much simpler than going and manually selecting each artwork downloading it and then applying it to steam. I have also swapped out the images with Prism Launcher ones rather than the legacy Polymc screenshots that were still there. This closes #35 and #42

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 _site/
 node_modules/
 .cache/
+.vscode/
```
Comment (txtsd) [inline .gitignore:4]: Insert newline at EOF pls

Comment (Lutitious) [reply .gitignore]: Done

File: src/download/steam-deck.md [hunk 2]
```diff
@@ -38,39 +38,37 @@ Now Prism Launcher should be in your Steam Library.
...
-Now, navigate to where you have saved the images, and select the background. I saved them in my Downloads folder, which will have the path of `/home/deck/Downloads`.
-Then do the same thing for `Set Custom Logo`, and select the logo.
+The easiest way to setup artwork on the steam deck is to use [SteamGridDBBoop](https://www.steamgriddb.com/boop).
+You can check out [Prism Launcher's SteamGridDB page](https://www.steamgriddb.com/game/5359069) for our official artwork, as well as community-provided artwork.
+Alternatively, there also is a [SteamGridDB page of Minecraft](https://www.steamgriddb.com/game/38365).
+This page contains a lot of artwork for Minecraft, which can be used for Prism Launcher as well.
 
-Additionally, you can change the **cover-art** (also called "grid" on SteamGridDB).
-There are two types of cover-art, one is the normal vertical one, and the other is the wider one.
+Firstly install Steamgriddb Boop on your Steam Deck via the Discover software center.
```
Comment (txtsd) [inline src/download/steam-deck.md:56]: `s/Steamgriddb/SteamGridDB/g`

Comment (txtsd) [reply src/download/steam-deck.md:56]: I can approve after that.

Comment (Lutitious) [reply src/download/steam-deck.md]: Done

**Comment by txtsd** [comment]
Next time, create a separate branch before making changes. Then you can PR from that branch instead of master/main.

**Comment by Lutitious** [comment]
Got it will do

**Pull request merged**